### ISSUE TITLE
Stagger the reload (configurable)

### ISSUE THREAD

**Issue by bassmanitram** [mohamedmansour/reload-all-tabs-extension#8]
If there are a lot of tabs open, a reload could overload the network (and my work network is notoriously slow and can get nasty if too many requests hit it from the same source).

There probably IS a "stagger" functionality already built in (I came across this issue in my own extension years ago), but having it configurable (max-at-a-time for example) could be useful.

Excellent "I say what I do, I do what I say" extension, though!

**Comment by mohamedmansour** [comment]
Hi @bassmanitram if I understand correctly, do you want a feature where loading is sequential instead of in parallel? And add a delay after each load?

**Comment by bassmanitram** [comment]
If possible - but it's not urgent and the extension is otherwise exactly
what I needed :)

**Comment by matthewhively** [comment]
This would be nice to have for sure.
